src/Compilers/Core/Portable/Compilation/Compilation.cs

```diff
@@ -2416,7 +2423,8 @@ internal void EnsureAnonymousTypeTemplates(CancellationToken cancellationToken)
         /// <param name="pdbStream">Stream to which the compilation's debug info will be written.  Null to forego PDB generation.</param>
         /// <param name="xmlDocumentationStream">Stream to which the compilation's XML documentation will be written.  Null to forego XML generation.</param>
         /// <param name="win32Resources">Stream from which the compilation's Win32 resources will be read (in RES format).
-        /// Null to indicate that there are none. The RES format begins with a null resource entry.</param>
+        /// Null to indicate that there are none. The RES format begins with a null resource entry.
+        /// Note that the caller is responsible for disposing this stream, if provided.</param>
```

Comment:
>Note that the caller is responsible for disposing this stream, if provided. [](start = 12, length = 75)

Is this true of all the `Stream` parameters to this method? If I was using this method, I'd be surprised if that wasn't the case, and this comment might make me wonder about the other streams.

Reply:
Only for *input* streams. I believe we will close output streams such as peStream, pdbStream, xmlDoc, sourceLink, etc.